creator-node/src/FileProcessingQueue.js

```diff
@@ -35,26 +35,25 @@ class FileProcessingQueue {
       }
     )
 
-    this.queue.process(
-      PROCESS_NAMES.transcode,
-      MAX_CONCURRENCY,
-      async (job, done) => {
-        const { transcodeParams } = job.data
-
-        try {
-          const response = await this.monitorProgress(PROCESS_NAMES.transcode, transcodeFn, transcodeParams)
-          done(null, { response })
-        } catch (e) {
-          this.logError(transcodeParams.logContext, `Could not process taskType=${PROCESS_NAMES.transcode} uuid=${transcodeParams.logContext.requestID}: ${e.toString()}`)
-          done(e.toString())
-        }
-      })
+    this.queue.process(PROCESS_NAMES.transcode, MAX_CONCURRENCY, async (job, done) => {
+      const { transcodeParams } = job.data
+
+      try {
+        const response = await this.monitorProgress(PROCESS_NAMES.transcode, transcodeFn, transcodeParams)
+        done(null, { response })
+      } catch (e) {
+        this.logError(transcodeParams.logContext, `Could not process taskType=${PROCESS_NAMES.transcode} uuid=${transcodeParams.logContext.requestID}: ${e.toString()}`)
+        done(e.toString())
+      }
+    })
+
+    this.getFileProcessingQueueJobs = this.getFileProcessingQueueJobs.bind(this)
```

Comment:
this has to be outside the function. i would put this after line 117 after getFileProcessingQueueJobs

Reply:
Not sure what this means. I think the bind should be where it is in the constructor. The diff above is just a formatting change.

Reply:
Oh sorry I closed the parens wrong mentally 😅. You're right, it's in the right place